test_test.go

```diff
@@ -25,5 +29,27 @@ func TestGitCollector(t *testing.T) {
 	require.NoError(t, gitCollectorTest.Prepare())
 	require.NoError(t, gitCollectorTest.RunLoad())
 
-	gitCollectorTest.GetResults()
+	require.True(t, gitCollectorTest.GetResults())
+}
+
+// TestMetadataRetrieval
+// runs regression comparison for remote:master and the first release
+// <expected> no errors occurred during tests execution
+// <expected> GetResults returns true
+func TestMetadataRetrieval(t *testing.T) {
+	config := regression.NewConfig()
+	config.BinaryCache = "binaries"
+	config.Versions = []string{"remote:master", "v0.1.0"}
```

Comment:
afaik `{"remote:regression", "remote:master"}` order is required

reminder: when `regression.yml` will be added both branches should contain it, so you would possibly need to create 
`regression` branch

Reply:
For this I will switch over the conversation to slack.

Reply:
I will change the order once everything is created.